base_phone/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Base Phone',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
Now with the OCA Bot Merger, there's no need to manually bump version. Even more, in this case it would be the minor version to change.